### PR TITLE
Functional canlendar UI with email form

### PR DESCRIPTION
**Pull request by JoshL579**

### CODE REVIEW

File: client/src/components/EmailDialog.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        fetch('/appointment', {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            credentials: "include",
+            body: JSON.stringify({
+                dateTime: dateTimeString,
+                timezone: timezone,
+                email: email
```
Comment (JoshL579) [inline client/src/components/EmailDialog.js:93]: - dateTimeString format: `2021-03-28T11:11:53.859Z`
- timezone format: `420` in minute, need to /60 for hours. But might need to change to other formats depends on usage.

**Comment by JoshL579** [comment]
image
image

File: client/src/components/EmailDialog.js (hunk 1 above)
Comment (fangyu3) [reply client/src/components/EmailDialog.js:93]: Needs to add the person's name. 

Also need a way to figure out which type of event was booked. 

Also, did we ever finish the event link functionality? i.e. clicking on url to get to the calendar page?

**Review by fangyu3 — APPROVED** [review]
Branch looks good! Feel free to merge. 

Left some comments not related to the branch but regarding the entire appointment booking flow.

File: client/src/components/EmailDialog.js (hunk 1 above)
Comment (JoshL579) [reply client/src/components/EmailDialog.js:93]: the copy link button on the cards in dashboard is working, but we need to figure out a way to make the url work for open a unique calendar.

**Pull request merged**